### PR TITLE
Create verify-azure-issue.yml

### PR DESCRIPTION
**Pull request by eccles**
AB#8391

### CODE REVIEW

File: .github/workflows/verify-azure-issue.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        id: 'get-pr-commits'
+        uses: tim-actions/get-pr-commits@master
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Check Subject Line Length
+        uses: tim-actions/commit-message-checker-with-regex@v0.3.1
+        with:
+          commits: ${{ steps.get-pr-commits.outputs.commits }}
+          pattern: '^.{0,75}(\n.*)*$'
```
Comment (robinbryce) [inline .github/workflows/verify-azure-issue.yml:22]: 75 is to short. Enforcing this at all is counter productive. 128 I can live with

**Review by robinbryce — CHANGES_REQUESTED** [review]

File: .github/workflows/verify-azure-issue.yml (hunk 1 above)
Comment (eccles) [reply .github/workflows/verify-azure-issue.yml]: fixed at 128

File: .github/workflows/verify-azure-issue.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        uses: tim-actions/get-pr-commits@master
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Check Subject Line Length
+        uses: tim-actions/commit-message-checker-with-regex@v0.3.1
+        with:
+          commits: ${{ steps.get-pr-commits.outputs.commits }}
+          pattern: '^.{0,128}(\n.*)*$'
+          error: 'Subject too long (max 75)'
```
Comment (robinbryce) [inline .github/workflows/verify-azure-issue.yml:23]: max 75 is now wrong. I would put the recomendation in the error too:

'Subject too long, recomended is under 50 (max 128)'

**Review by robinbryce — APPROVED** [review]

File: .github/workflows/verify-azure-issue.yml (hunk 2 above)
Comment (eccles) [reply .github/workflows/verify-azure-issue.yml]: fixed

**Review by robinbryce — APPROVED** [review]

**Pull request merged**